test/proxy.js

```diff
@@ -155,6 +155,7 @@ const opts = /** @type {Partial<TalkbackOptions>} */ ({
   port,
   path: './test/tapes',
   record: recordMode,
+  silent: true,
```

Comment:
Interesting, should this be configurable with an env var? I sometimes forgot to update tapes. No need to be addressed here but I'm just noting it.